operator/v1/types_ingress.go

```diff
@@ -1306,6 +1306,23 @@ type IngressControllerTuningOptions struct {
 	// +kubebuilder:validation:Format=duration
 	// +optional
 	TLSInspectDelay *metav1.Duration `json:"tlsInspectDelay,omitempty"`
+
+	// healthCheckInterval defines how long the router waits between two consecutive
+	// health checks on its configured backends.  This value is applied globally as
+	// a default for all routes, but may be overridden per-route by the route annotation
+	// "router.openshift.io/haproxy.health.check.interval".
+	//
+	// Setting this to less than 5s can cause excess traffic due to too frequent
+	// TCP health checks and accompanying SYN packet storms.  Alternatively, setting
+	// this too high can result in increased latency, due to backend servers that are no
+	// longer available, but haven't yet been detected as such.
+	//
+	// If unset, the default healthCheckInterval is 5s.
```

Comment:
As currently phrased, this default is an API promise. Is that your objective or do you prefer to have the platform choose a default which can change over time?

Reply:
We don't have a requirement or a request to make the healthCheckInterval dependent on the platform.  This defautl applies to the HAProxy on any platform.

Reply:
"the platform" in that sentence meant OCP.  Do we need/want the flexibility to choose a different default value at some point in the future?

Reply:
Updated comment as discussed.